analyses/cell-type-consensus/scripts/prepare-cell-type-ontologies.R

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env Rscript
+
+# This script is used to match cell types present in PanglaoDB to cell ontology IDs
+# The output is a TSV file that contains `ontology_id`, `human_readable_value`, and `panglao_cell_type`
+
+# If the output file already exists any cell types with assigned ontology IDs will not be modified 
+# Only cell types without ontology IDs will be used when matching to avoid any NA replacements 
+
+# File paths -------------------------------------------------------------------
+module_base <- rprojroot::find_root(rprojroot::is_renv_project)
+
+# read in original ref file 
+ref_file <- file.path(module_base, "references", "PanglaoDB_markers_2020-03-27.tsv")
```

Comment:
A few thoughts:

- Do we want to include a data download script that grabs this from `scpca-nf` (provided I interpreted your comment I include below correctly) in this module?
- Should we explicitly ignore this file in this module?

I think that's how I'd address this concern:

> I did want to point out that currently the actual reference file from Panglao is not in the repo because it's ~ 1000 KB and we have a 200 KB limit on TSV files. How do we want to proceed here? We probably don't really need it and I could just make a list of the cell types and save as a text file to read in and point to where the file lives in [`scpca-nf`](https://github.com/AlexsLemonade/scpca-nf/blob/main/references/PanglaoDB_markers_2020-03-27.tsv) or we could just make an exception for this file?